py3dtilers/TilesetReader/TilesetReader.py

```diff
@@ -0,0 +1,129 @@
+import sys
+
+from py3dtiles import TilesetReader
+from .parsed_b3dm import ParsedB3dms
+from .tile_hierarchy import TileHierarchy
+from ..Common import Tiler, create_tileset
+
+
+class B3dmTiler(Tiler):
```

Comment:
Semantics again: maybe naming that class `3DTilesImporter` would clarify the fact that this class is inherently a treatment ? (just as converters are: refer above)